api/app/src/external/commonwell/document/document-query-sandbox.ts

```diff
@@ -25,12 +26,18 @@ export async function sandboxGetDocRefsAndUpsert({
 }): Promise<DocumentReference[]> {
   const { log } = Util.out(`sandboxGetDocRefsAndUpsert - M patient ${patient.id}`);
 
+  await Util.sleep(3000);
+
   const patientData = getSandboxSeedData(patient.data.firstName);
   if (!patientData) return [];
 
   const entries = patientData.docRefs;
   log(`Got ${entries.length} doc refs`);
 
+  let convertibleDocCount = patientData.docRefs.filter(
+    docRef => docRef.docRef.content?.[0]?.attachment?.contentType === "application/xml"
+  ).length;
```

Comment:
should we make a function that is used in all places where we consider a doc ref convertible?

this won't work since it's missing `text/xml`

maybe use something like this? https://github.com/metriport/metriport/blob/d178652349229e8979124d70e047da337a1e51a7/api/app/src/external/fhir-converter/converter.ts#L29-L32